cookbook_se/Dockerfile.sagemaker

```diff
@@ -15,9 +15,6 @@ RUN : \
     && apt-get update \
     && apt-get install -y python3.8 python3-pip python3-venv make build-essential libssl-dev curl vim
 
-# This is necessary for opencv to work
```

Comment:
was the flytefile example already changed to use the opencv enabled image?

Reply:
Oh this is just the sagemaker one... nm

Reply:
this is .sagemaker dockerfile, we dont use opencv anymore